### PR TITLE
Bundle latest 2.2 and 3.1 site extension packages

### PR DESCRIPTION
**Pull request by dougbu**
- part of dotnet/aspnetcore-internal#3676

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -241,8 +241,8 @@
     <SystemIdentityModelTokensJwtPackageVersion>6.7.1</SystemIdentityModelTokensJwtPackageVersion>
     <!-- Packages from 2.1, 2.2, and 3.1 branches used for site extension build. -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21PackageVersion>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>3.1.7-servicing-20372-13</MicrosoftAspNetCoreAzureAppServicesSiteExtension31PackageVersion>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>2.2.5</MicrosoftAspNetCoreAzureAppServicesSiteExtension22PackageVersion>
```
Comment (dougbu) [inline eng/Versions.props:244]: This won't _hurt_ will it @BrennanConroy❔

Comment (BrennanConroy) [reply eng/Versions.props:244]: No, but also no point :D

Comment (dougbu) [reply eng/Versions.props:244]: To clarify: Do you want this line backed out❔

Comment (BrennanConroy) [reply eng/Versions.props:244]: I don't care either way 😄

Comment (dougbu) [reply eng/Versions.props:244]: Then please approve😺

**Review by BrennanConroy — APPROVED** [review]

**Review by wtgodbe — APPROVED** [review]

**Pull request merged**